pkg/controller/volume/events/event.go

```diff
@@ -30,4 +30,5 @@ const (
 	ProvisioningCleanupFailed = "ProvisioningCleanupFailed"
 	ProvisioningSucceeded     = "ProvisioningSucceeded"
 	WaitForFirstConsumer      = "WaitForFirstConsumer"
+	ExternalExpanding         = "ExternalExpanding"
```

Comment:
I think `ExternalExpanding` is bit vague. Can we rename this to something like `VolumeResizePending` or `VolumeExpansionPending` ? Once a external controller picks up the PVC it can go ahead and finish the process.

Reply:
`ExternalProvisioning` is the same, I prefer ExternalExpanding for consistency. VolumeResizePending wouldn't tell you kubernetes is just waiting for an external actor to do something, so the user wouldn't know who to blame

Reply:
Yeah I see your point. It can go either way. I am not sure if `ExternalProvisioning` name makes sense either. Kubernetes can not know for sure that a volume is being externally provisioned or resized. All it knows is - it can't do this itself.

Reply:
I also went for consistency. This is how events stack up for externally provisioned volumes after my changes:
```
Events:
  Type       Reason                Age   From                         Message
  ----       ------                ----  ----                         -------
  Normal     ExternalProvisioning  44s   persistentvolume-controller  waiting for a volume to be created, either by external provisioner "netapp.io/trident" or manually created by system administrator
  Normal     ProvisioningSuccess   36s   netapp.io/trident            Kubernetes frontend provisioned a volume and a PV for the PVC.
  Normal     ExternalExpanding     14s   volume_expand                Ignoring the PVC: didn't find a plugin capable of expanding the volume; waiting for an external controller to process this PVC.
Mounted By:  <none>
```